pwa/src/styles/main.css

```diff
@@ -219,3 +219,24 @@ a:hover {
     border: none !important;
     background: inherit !important;
 }
+
+.spinner-overlay {
+    height: 100%;
+    width: 100%;
+    display: block;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    background-color: rgb(0,0,0);
+    background-color: rgba(0,0,0, .3);
+}
+
+.spinner-overlay-content {
+    position: relative;
+    top: 25%;
+    width: 100%;
+    text-align: center;
+    margin-top: 30px;
+}
```

Comment:
This is not centered right now, and is using multiple `top` variables. 

A easy fix for this would be to do the following:

1. Remove any and all styling from `.spinner-overlay-content` (therefore you can also remove the class itself in the component)
2. Add the following styles to `.spinner-overlay`:

```
display: flex; // so not block
justify-content: center; // makes it center horizontally
align-items: center; // makes it center vertically
```

Reply:
[CSS Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a widely accepted new standard for positioning elements (and is now also supported by all major browsers). Take some time to read some documentation, we will be using it a lot 😄